### PR TITLE
Remove `grunt test` and `grunt karma:all`

### PR DESCRIPTION
**Pull request by joshlory**
These don't currently pass on Staging. Remove them so devs don't expect them to pass.

Failing `node --max_old_space_size=4096 ./node_modules/.bin/grunt test` on Staging: https://gist.github.com/joshlory/2925bf0cadd09413b5c42fa5758b9766

### CODE REVIEW

**Comment by joshlory** [comment]
screen shot 2017-08-02 at 9 02 58 pm

**Comment by islemaster** [comment]
Instead of nuking `grunt test` can we alias it to do the same thing a `test-low-memory`... run the suites in sequence?  Then maybe we can just get rid of `test-low-memory` and that becomes that canonical way to run all our JavaScript tests.

**Comment by Bjvanminnen** [comment]
`npm run test` currently calls `grunt test`. If we do remove `grunt test` we should still have `npm run test` do something - maybe lint and test-low-memory?

**Comment by joshlory** [comment]
I think it makes sense to keep `npm run test` and remove `grunt test`.  I've updated the PR.  Any objections?

**Comment by davidsbailey** [comment]
Has any investigation been done as to why karma:all is not passing? The "Attempt to wrap foo because it is already wrapped" errors seem to indicate problems that if we choose to bury them now may surface again when our test ordering changes.

**Comment by joshlory** [comment]
I haven't investigated root cause for the failures.  This PR was motivated by trying to track down failures of `grunt test` on my branch, only to realize they also fail on Staging.

We could instead work on making them pass, and adding CI coverage so that they keep passing?  The issue is that all the tests run in a single browser instance, so any globals that are leaked can accidentally be relied on in another test.

**Comment by davidsbailey** [comment]
OK, that seems reasonable. If we aren't running the `karma:all` target regularly, then you're right that we shouldn't keep it around.

File: apps/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "lint": "eslint --ext .js,.jsx src test",
-    "test": "npm run lint && grunt test",
+    "test": "./test-low-memory.sh",
     "test-low-memory": "./test-low-memory.sh",
```
Comment (islemaster) [inline apps/package.json:13]: Next step (after this pr): Replace everything that calls `(npm/yarn) run test-low-memory` with `yarn test`

**Pull request merged**